app/facades/book_facade.rb

```diff
@@ -1,29 +1,17 @@
 class BookFacade
 
-  def self.get_all_books(user_id)
-    json = BooksApiServices.get_users_books(user_id)
+  def self.search_for_books(title)
+    json = BooksApiServices.get_search_books(title)
 
-    books = json.each do |book_attributes|
-      Book.new(book_attributes)
-    end
-    #feel this may be a good start, but right now with too much unknown as to the json response, this may need retooling
-  end
-
-  def self.get_single_book(user_id, book_id)
-    json = BooksApiServices.get_users_books(user_id)
-
-    book_attributes = json.find do |book|
-      book[:id] = book_id
+    json.map do |book_attributes|
+      BookPoro.new(book_attributes)
     end
-
-    Book.new(book_attributes)
-    #for right now this seems the easiest way to do this, but we could send something else over
   end
 
-  def self.get_books_with_title_filter(user_id, title_params)
-    json = BooksApiServices.get_users_books_with_title_filter(user_id, title_params)
+  def self.get_single_book(volume_id)
```

Comment:
just to clarify (maybe a name update?) - this gets a book that is NOT in your library yet, right? and then a separate method would get a book that IS in your library and include the words?

Reply:
all single books would pull from the same source.